addons/mail/static/src/messaging/component/notification_list/notification_list.js

```diff
@@ -89,6 +90,18 @@ class NotificationList extends Component {
                     uniqueId: thread.localId,
                 };
             });
+        if (props.filter === 'all') {
+            var notificationGroups = this.env.messaging.notificationGroupManager.groups;
+            notifications = Object.values(notificationGroups)
+                .sort((group1, group2) => {
+                    return group1.date.isAfter(group2.date) ? -1 : 1;
```

Comment:
inline

Reply:
inline what?

Reply:
`.sort((group1, group2) => group1.date.isAfter(group2.date) ? -1 : 1))`

Could be in multi-line... By "inline" I meant arrow function with one-liner notation (so no scope with `{ return ... }`).

Reply:
ok for no return, still need multiline if we follow 80 limit